### PR TITLE
31367 GDPR (checkbox) Field not recognised

### PR DESCRIPTION
**Pull request by dai-eastgate**
related to https://github.com/onOffice-Web-Org/oo-wp-plugin/discussions/454

changed log :
add module special field

### CODE REVIEW

File: plugin/Renderer/InputFieldButtonAddRemoveRenderer.php [hunk 1]
```diff
@@ -46,6 +52,46 @@ public function __construct($name, $value)
...
     }
 
+	/**
+	 *
+	 * @param string $key
+	 * @param FieldsCollection $pFieldsCollection
+	 * @return string
+	 */
+
+	public function getModule(string $key, FieldsCollection $pFieldsCollection): string
```
Comment (andernath) [inline plugin/Renderer/InputFieldButtonAddRemoveRenderer.php:62]: Is public visibility necessary?

File: plugin/Renderer/InputFieldButtonAddRemoveRenderer.php [hunk 2]
```diff
@@ -46,6 +52,46 @@ public function __construct($name, $value)
...
+ 
+ 
+	 /**
+	  *
+	  * @return FieldsCollection
+	  *
+	  * @throws Exception
+	  */
+ 
+	public function buildFieldsCollection(): FieldsCollection
```
Comment (andernath) [inline plugin/Renderer/InputFieldButtonAddRemoveRenderer.php:75]: Is public visibility necessary?

**Review by andernath — CHANGES_REQUESTED** [review]
@dai-eastgate Could you please take a look inside codereview?

**Comment by dai-eastgate** [comment]
@andernath Please check this PR also :)

**Review by andernath — APPROVED** [review]
Thank you :)

**Pull request merged**